queries.yaml

```diff
@@ -77,7 +92,6 @@ layers:
     area-inclusion-threshold: 1
   roads:
     template: roads.jinja2
-    start_zoom: 5
```

Comment:
Let's followup in https://github.com/tilezen/vector-datasource/issues/1353 after this PR is merged.